packages/@okta/vuepress-site/docs/reference/rest/index.md

```diff
@@ -0,0 +1,360 @@
+    Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?node-id=3236%3A31016  postman_environment_quick_look_edit
+    -->
+
+1. In the environment tab, do the following:
+    1. Click the environment name, delete the placeholder text, and name your environment. For example: `dev-1234567 Okta Org`
+    {style="list-style-type:lower-alpha"}
+    1. For the `url` variable, in the **Initial Value** and **Current Value** columns, replace the placeholder text with your org's full URL. For example: `https://dev-1234567.okta.com`. Remember to remove the `-admin` part of your subdomain.
+
+1. Click **Save** near the top of the tab.
+1. To close the environment tab, hover over the tab and click the **x**.
+
+## Import a collection
+
+Go to the Okta [Postman collections](https://developer.okta.com/docs/reference/postman-collections/) reference to import the collection you want to test.
+
+> **Note:** Alternatively, you can fork a collection from the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview).
+
+To import a collection from the Okta [Postman Collections](https://developer.okta.com/docs/reference/postman-collections/) reference page:
+
+1. Click **Run in Postman** next to the collection you want.
+
+   For example, click this **Run in Postman** action to import the Users API collection:
+   [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9daeb4b935a423c39009)
+   > **Note:** The **Run in Postman** option is also available on each [core API reference](/docs/reference/core-okta-api/) page on this site.
+
+1. On the **Run in** dialog, either select to import your collection to your local Postman app or to your web Postman account.
+
+   If you select the web option, you need to sign in to your online Postman account.
+
+1. Select the workspace to import your collection, and then click **Import**.
+
+## Set up Okta for API access
+
+To access Okta APIs from Postman, you need to authenticate with the Okta API resource server. Okta APIs support the OAuth 2.0 authentication scheme that uses access tokens. Access tokens enable the bearer to perform specific actions on specific Okta endpoints, defined by the scopes in the token.
+
+Scoped access tokens have several advantages, including:
+
+* More access granularity
+* Shorter token lifespans
+* Can be generated and retrieved using an API
+
+> **Note:** Okta doesn't recommend using the Okta-propriety `SSWS` API token authentication scheme.
+> This API token scheme allows you to access a broad range of APIs since there's no scope associated with the token. Access to the APIs depends on the privileges of the user that [created the API token](/docs/guides/create-an-api-token/main/). The API token also has a fixed expiry date.
```

Comment:
```suggestion
> This API token scheme allows you to access a broad range of APIs because there's no scope associated with the token. Access to the APIs depends on the privileges of the user that [created the API token](/docs/guides/create-an-api-token/main/). The API token also has a fixed expiry date.
```